content/en/docs/concepts/configuration/secret.md

```diff
@@ -95,11 +95,16 @@ Kubernetes imposes on them.
 | `bootstrap.kubernetes.io/token` | bootstrap token data |
 
 You can define and use your own Secret type by assigning a non-empty string as the
-`type` value for a Secret object. An empty string is treated as an `Opaque` type.
+`type` value for a Secret object (an empty string is treated as an `Opaque` type).
```

Comment:
Don't see a reason for this change

Reply:
In spoken English, the speaker might shift tone slightly to indicate that the phrase “an empty string is treated as an Opaque type” is an associated details. In written English, the parens [are my attempt to] convey that.